ui/app/components/wizard/features-selection.js

```diff
@@ -63,20 +65,23 @@ export default Component.extend({
         ],
         selected: false,
         show: true,
+        permission: 'policies',
```

Comment:
Using this, I think overall the permissions checks for this should probably be more granular and maybe even check specific actions on various paths (in this case you need to be able to create, edit, and delete acl policies to complete the tutorial).